Zend/tests/throw/leaks.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+throw expression should not leak temporaries
+--FILE--
+<?php
+
+try {
+    new stdClass(throw new Exception);
+} catch (Exception $e) {
+    echo "Caught\n";
+}
+
+try {
+    $a = [];
+    ($a + [1]) + throw new Exception;
+} catch (Exception $e) {
+    echo "Caught\n";
+}
+
+try {
+    @throw new Exception;
+} catch (Exception $e) {
+    echo "Caught\n";
+}
+var_dump(error_reporting());
+
+?>
+--EXPECT--
+Caught
+Caught
+Caught
+int(32767)
```

Comment:
Could this use the following to override the users php.ini settings instead, to avoid test failures when a tester already installed something to the configured `--prefix`?

```
--INI--
error_reporting=E_ALL
```

Reply:
This is already enforced by run-tests.

Reply:
See https://github.com/php/php-src/blob/d5e51fc186815f8132beb4ceed9f16c91b32a493/run-tests.php#L236 for list of ini settings that run-tests overwrites.